src/mainwindow.cpp

```diff
@@ -4096,6 +4096,21 @@ void MainWindow::on_actionExportEDL_triggered()
 void MainWindow::on_actionExportFrame_triggered()
 {
     if (!MLT.producer() || !MLT.producer()->is_valid()) return;
+    if (Settings.proxyEnabled()) {
+        QMessageBox dialog(QMessageBox::Question,
+                           tr("Export from from proxy"),
+                           tr("This project is currently using proxy files. The exported frame will be taken from the lower resolution proxy instead of the original source.\n\n"
+                              "Do you still want to continue?"),
+                           QMessageBox::No | QMessageBox::Yes,
+                           this);
+        dialog.setDefaultButton(QMessageBox::Yes);
+        dialog.setEscapeButton(QMessageBox::No);
+        dialog.setWindowModality(QmlApplication::dialogModality());
+        int result = dialog.exec();
+        if (result == QMessageBox::No) {
```

Comment:
By checking for No, please be aware that closing the dialog via window title button will not return early here, and that could be unexpected.

Reply:
Fixed